src/entities/course/ui/course-card/course-card.tsx

```diff
@@ -48,12 +57,33 @@ export const CourseCard = ({
         <Subtitle fontSize="small">{title}</Subtitle>
       </div>
       <div className={cx('course-info')}>
-        <DateLang
-          startDate={startDate}
-          registrationEndDate={registrationEndDate}
-          language={language}
-          mode={mode}
-        />
+        {!showMentoringStartDate && (
+          <ShortInfoPanel
+            startDate={startDate}
+            registrationEndDate={registrationEndDate}
+            language={language}
+            mode={mode}
+          />
+        )}
+        {showMentoringStartDate && (
+          <div>
+            <DateSimple
+              label={LABELS.MENTOR_ACTIVITIES}
+              labelSeparator={
+                personalMentoringStartDate ? LABELS.MENTORING_DATES_SEPARATOR : TO_BE_DETERMINED
+              }
```

Comment:
Should it be `personalMentoringStartDate && personalMentoringEndDate ? ...`?
Also, I'm not sure `labelSeparator` prop is the correct place to handle rendering `TBD`.